tasks/Node/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "azure-pipelines-dtl-tasks",
+  "version": "3.0.0",
+  "description": "Collection of cross-platform Azure Pipelines tasks to interact with Azure DevTest Labs.",
+  "main": "index.js",
+  "dependencies": {
+    "@azure/arm-devtestlabs": "^3.2.0",
+    "@azure/arm-resources": "^1.1.0",
+    "@azure/ms-rest-js": "^2.0.4",
+    "@azure/ms-rest-nodeauth": "^2.0.3",
+    "@azure/storage-blob": "^10.4.0",
+    "azure-pipelines-task-lib": "^2.8.0",
+    "isomorphic-fetch": "^2.2.1",
+    "minimist": "^1.2.0",
+    "tough-cookie": "^3.0.1",
+    "uuid": "^3.3.2"
+  },
+  "devDependencies": {
+    "@types/mocha": "^5.2.7",
+    "@types/ncp": "^2.0.1",
+    "@types/node": "^10.14.14",
+    "@types/q": "^1.5.2",
+    "ncp": "^2.0.0",
+    "react-dev-utils": "^9.0.1",
+    "rimraf": "^2.6.3",
+    "ts-loader": "^6.0.4",
+    "typescript": "^3.0.3",
+    "webpack": "^4.39.0"
+  },
+  "scripts": {
+    "build-dev": "npm run copy-dev && tsc",
```

Comment:
Why is this command invoking the typsecript compiler directly? Aren't all of your TS files built by webpack now?

Reply:
When testing locally, we do need to run `tsc`. So, when setting things up for local we do use it independent of packaging. We only use Webpack for packaging.